assets/stylesheets/components/_content-menu.scss

```diff
@@ -0,0 +1,26 @@
+.content-menu { 
+  background-color: $cr-tan-lighter; 
+  border: 1px solid $cr-gray-lighter!important; 
+  height: auto; 
+  margin-left: 10px;
+  padding-top: 10px; 
+}
+
+.dashed {
+  border-bottom: 1px dashed $cr-gray-lighter;
+  height: 1px;
+  opacity: 0.75; 
+  width: 85%;
+  margin-left: 24.5px; 
+  margin-bottom: 45px; 
+}
+
+.menu-link { 
+  position: relative; 
+  top: 4px; 
+}
+  
+.menu-title { 
```

Comment:
Could you nest this under `'content-menu` or change it to `.content-menu-title`. So we do not have any collisions since this is a somewhat generic name.